src/visibility.js

```diff
@@ -177,6 +191,45 @@ class Visibility extends BasePlugin {
     this._observer = null;
     this.eventManager.destroy();
   }
+
+  _dragMouseDown(e: MouseEvent) {
+    e.preventDefault();
+    // get the mouse cursor position at startup:
+    this._currMousePos.x = e.clientX;
+    this._currMousePos.y = e.clientY;
+    this.eventManager.listen(document, 'mouseup', this._dragStop.bind(this));
+    this.eventManager.listen(document, 'mousemove', this._floatingDrag.bind(this));
+  }
+
+  _floatingDrag(e: MouseEvent) {
+    if (this._throttleWait) return;
+    e = e || window.event;
+    e.preventDefault();
+    // calculate the new cursor position:
+    const deltaMousePosX = this._currMousePos.x - e.clientX;
+    const deltaMousePosY = this._currMousePos.y - e.clientY;
+    this._currMousePos.x = e.clientX;
+    this._currMousePos.y = e.clientY;
+    const floatingContainer = this._floatingContainer; // flow
+    // set the element's new position
+    if (floatingContainer) {
+      const boundClientRect = floatingContainer.getBoundingClientRect();
+      floatingContainer.style.top = boundClientRect.top - parseInt(floatingContainer.style.marginTop) - deltaMousePosY + 'px';
+      floatingContainer.style.left = boundClientRect.left - parseInt(floatingContainer.style.marginLeft) - deltaMousePosX + 'px';
+    }
+
+    // handle throttling to avoid performance issues on dragging
+    this._throttleWait = true;
+    setTimeout(() => {
+      this._throttleWait = false;
+    }, 30);
```

Comment:
please move magic numbers to const in top of file.
Do we need to set this number as configurable?

Reply:
This is the miliseconds of the delay. Meaning the the mouse move will hit no more than ~30 times a second.
I will take out as const - don't think it is needed as a config